config/webpack/partial/manifest.js

```diff
@@ -0,0 +1,29 @@
+"use strict";
```

Comment:
@jchip will do, I mentioned that same thing here: https://github.com/electrode-io/electrode-archetype-react-app/pull/46#discussion_r87057797